Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUBufferImpl.cpp

```diff
@@ -45,11 +45,20 @@ BufferImpl::~BufferImpl()
     wgpuBufferRelease(m_backing);
 }
 
+static Size64 getSizeFromBacking(WGPUBuffer backing, std::optional<Size64> size, Size64 offset)
+{
+    if (size.has_value())
+        return size.value();
```

Comment:
So, is `size` is a value, it doesn't get `getSizeFromBacking`? How about `getMappedSize(m_backing, size, offset)`